apps/communications/contacts/test/marionette/list_test.js

```diff
@@ -51,6 +51,68 @@ marionette('Contacts > List', function() {
     });
   });
 
+  suite('List Order > ', function() {
+    var data = [['GG', 'E'], ['AA', 'Z'],
+      ['XX', 'C'], ['CC', 'X'], ['EE', 'G'], ['FF', 'F'], ['HH', 'D'],
+      ['BB', 'Y'], ['YY', 'B'], ['ZZ', 'A'], ['DD', 'H']];
+    setup(function() {
+      data.forEach(contactData => {
+        subject.addContact({
+          givenName: contactData[0],
+          familyName: contactData[1]
+        });
+      });
+    });
+
+    function getNamesInList() {
```

Comment:
This function would be better in `contacts.js`. We also don't need to inject some Gaia JS:
```js
Selectors: {
   contactItems: "li:not([data-group="ice"]).contact-item p"
}
// ...

get contacts() {
  var contactElements = this.client.findElements(Selectors.contactItems);
  return contactElements.map(function(contactElement) { return contactElement.text() });
}
```